keps/sig-auth/2718-20210511-client-exec-proxy/README.md

```diff
@@ -663,9 +667,8 @@ _This section must be completed when targeting beta graduation to a release._
 * **What are the SLIs (Service Level Indicators) an operator can use to determine
 the health of the service?**
   - [ ] Metrics
-    - Metric name:
-    - [Optional] Aggregation method:
-    - Components exposing the metric:
+    - Metric name: `rest_client_exec_plugin_with_proxy_call_total`
```

Comment:
Is that just a count? can we differentiate errors? (non-blocking question)

Reply:
Yes, we will be able to differentiate errors, the implementation follow the example of: https://github.com/kubernetes/client-go/blob/d3b97581d2e22eef0f02a6667e19dfc193df7a3e/plugin/pkg/client/auth/exec/metrics.go